ld/STM32F429ZI.ld

```diff
@@ -245,4 +245,10 @@ SECTIONS
     PROVIDE(__heap_size = SIZEOF(.heap));
     PROVIDE(__mbed_sbrk_start = ADDR(.heap));
     PROVIDE(__mbed_krbs_start = ADDR(.heap) + SIZEOF(.heap));
+
+    /* Provide physical memory boundaries for uVisor. */
+    __uvisor_flash_start = ORIGIN(VECTORS);
+    __uvisor_flash_end = ORIGIN(FLASH) + LENGTH(FLASH);
+    __uvisor_sram_start = ORIGIN(CCM);
```

Comment:
Yes, this has to do with vector relocation. uVisor needs the **physical** memory boundary of the memories it uses for itself. These are:
* Flash + SRAM for the K64F
* Flash + CCM for the STM32F429

In the K64F you saw a `ORIGIN(SRAM) - 0x200` because the **ld** section called `SRAM` starts at offset 0x200 (vectors are relocated before that offset).

Here the relocation still happens (offset 0x1AC) but the uVisor doesn't care (it uses the CCM, which has no offset).

@niklas This will need to be captured by templating. That is, the `__uvisor_{flash, sram}_{start, end}` symbols refer to the physical memories uVisor actually uses, which is currently not encoded in any way.